README.md

```diff
@@ -15,10 +15,18 @@ ready.
 
 This framework runs on most Linux based systems.
 
-You need python 2.7 or greater.
+You need python 2.7 or greater and python-pip.
 
+    sudo apt-get install python python-pip
+
+You need pexpect.
+
+    sudo pip install pexpect
+    
 You will also need (recent) ipmiutil - 1.8.15 or above should be adequate.
 
+	sudo apt install ipmiutil
```

Comment:
I am sorry! I don't check this utility uses ipmiutil or ipmitool. I just add the install command by the description.